pkg/build/cmd/uploadpackages.go

```diff
@@ -110,6 +118,40 @@ func UploadPackages(c *cli.Context) error {
 	return nil
 }
 
+// Corner case for custom enterprise2 mode
+func bucketForEnterprise2(releaseModeConfig *config.BuildConfig, event string) (string, error) {
+	if event == config.Custom {
+		buildConfig, err := config.GetBuildConfig(config.ReleaseBranchMode)
+		if err != nil {
+			return "", err
+		}
+		return buildConfig.Buckets.ArtifactsEnterprise2, nil
```

Comment:
Shouldn't that part also return an error if ArtifactEnterprise2 isn't set?

Reply:
Hm, good catch but the `ArtifactEnterprise2` will be always there for `ReleaseBranchMode`. The test will break if something changes really.